src/main/java/de/hysky/skyblocker/mixins/InventoryScreenMixin.java

```diff
@@ -15,4 +20,10 @@ public abstract class InventoryScreenMixin {
     private RecipeBookWidget skyblocker$replaceRecipeBook(RecipeBookWidget original) {
         return SkyblockerConfigManager.get().general.itemList.enableItemList && Utils.isOnSkyblock() ? new ItemListWidget() : original;
     }
+
+    @WrapOperation(method = "init", at = @At(value = "NEW", target = "(IIIILnet/minecraft/client/gui/screen/ButtonTextures;Lnet/minecraft/client/gui/widget/ButtonWidget$PressAction;)Lnet/minecraft/client/gui/widget/TexturedButtonWidget;"))
```

Comment:
Could be a modify expression value

Reply:
something like 
@ModifyExpressionValue
public Button thing(Button original) {
button.setX(button.getX() + 21)
return button

Reply:
Actually I meant `ModifyArg`, same for the wrap operation below.